runtimes/scala/play-framework-2.md

```diff
@@ -80,11 +80,24 @@ Redirect to HTTPS With Play
 To be able to use `request.secure`, you have to add `trustxforwarded=true` in
 `application.conf`.
 
+## Multi-module project
+
+If you have a single repository with multiple modules, then you can specify which module to run with `CC_SBT_TARGET_DIR`.
+
+For instance, if your Sbt project contains a `shared` and `play` module and you want to execute the `play` module, then add
+`CC_SBT_TARGET_DIR=play` in the application's environment variables.
+
+
 ## Deploy on Clever Cloud
 
 Application deployment on Clever Cloud is via Git. Follow [these steps](/doc/clever-cloud-overview/add-application/) to deploy your application.
 
 
+## Bad root server path
```

Comment:
I would go with "Bad root server path error" as a title so it's understandable that we're talking about an error

Reply:
@gervaisb can you please just change that so I can merge your PR ?